### DISCUSSION TITLE
Sample code for new kumites

### DISCUSSION THREAD

**Discussion by Captain-Quack** [codewars/codewars.com#2925] [Ideas]
(*this is a standalone idea from [this](https://github.com/codewars/codewars.com/discussions/2888) list*)
It would be nice to have the same sample code generated for the example kata show up when making a kumite. It can be a significant pain to set up a lot of boilerplates a lot of times, especially for languages like COBOL and C#!

**Comment by hobovsky** [comment]
I re-raised this idea as an enhancement/feature request here: https://github.com/codewars/codewars.com/issues/2954 .

Additionally, I created this collection of [Authoring examples](https://www.codewars.com/collections/authoring-examples), which I use as a template/source of boilerplate for new translations I create.
